addons/event/models/event_event.py

```diff
@@ -496,6 +496,15 @@ def write(self, vals):
         res = super(EventEvent, self).write(vals)
         if vals.get('organizer_id'):
             self.message_subscribe([vals['organizer_id']])
+        if 'stage_id' in vals:
```

Comment:
Technically shouldn't it be integrated in create also ?

I would also move this code in its own method called by write. That way we avoid inline code for a side business requirement.